core/App/screens/Scan.tsx

```diff
@@ -36,6 +42,26 @@ const Scan: React.FC<ScanProps> = ({ navigation }) => {
     }
   }
 
+  useMemo(async () => {
+    const data = await AsyncStorage.getItem(DidShowCameraDisclosureLocalStorageKey)
```

Comment:
Do we want to use the reducer for this as we do with other global settings?

Reply:
Fixed.